frontend/src/components/ParticipantSearch.js

```diff
@@ -35,7 +34,7 @@ const ParticipantSearch = observer(() => {
       <Grid
         container
         component="form"
-        onSubmit={handleSubmit}
+        onSubmit={e => (handleSubmit(e), props.handleToggle())}
```

Comment:
Sorry this is the thrid time now. lets see if I can get this right. the `handleToggle` should just be moved into the handleSubmit function.